### ISSUE TITLE
Incomplete SpanContext type on npm

### ISSUE THREAD

**Issue by lykkin** [opentracing/opentracing-javascript#123]
It looks like the `SpanContext` class definition in the published version is empty (i.e. missing `toTraceId` and `toSpanId` methods).  Interestingly enough, when I build this locally the class definition that's produced looks proper.  Caching issue maybe?

published `span_context.d.ts`:
```
export declare class SpanContext {
}
export default SpanContext;
```

locally built `span_context.d.ts`:
```
export declare class SpanContext {
    /**
     * Returns a string representation of the implementation internal trace ID.
     *
     * @returns {string}
     */
    toTraceId(): string;
    /**
     * Returns a string representation of the implementation internal span ID.
     *
     * @returns {string}
     */
    toSpanId(): string;
}
export default SpanContext;
```

**Comment by rochdev** [comment]
This was added recently and not yet released.

@yurishkuro Any objection to release this?

**Comment by yurishkuro** [comment]
no objections, e.g. 0.15

**Comment by aimazh** [comment]
what's the approximate release date?

**Comment by chlab** [comment]
Just stumbled over this as well. Would be great to get a new version on npm that has these two methods.

**Comment by yurishkuro** [comment]
@bcronin do you have instructions on releasing this package?

**Comment by bcronin** [comment]
@yurishkuro , I added you as a package maintainer on npm.

As for instructions for releasing the package, the package looks to have been updated quite a bit since I last touched it 3 years ago so don't know if anything is needed beyond a version bump and `npm publish`.

**Comment by yurishkuro** [comment]
@bcronin  I see that you did  the last release:
```
3abef18 - 0.14.2 (1 year, 3 months ago) <bcronin>
```
It seems like there should be some "build" step before "publish".

**Comment by austinlparker** [comment]
I think it'd be `npm run prepare`, but that's currently giving an error at HEAD.

**Comment by austinlparker** [comment]
I've published 0.14.4 to npm which fixes this issue, thanks!
